components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue

```diff
@@ -185,11 +185,11 @@ const formatOfferStatus = (status: OfferStatusType, expiration: number) => {
 }
 
 const onWithdrawOffer = async (caller: string) => {
-  await submit(caller, ShoppingActions.WITHDRAW_OFFER, refetch)
+  await submit(caller, ShoppingActions.WITHDRAW_OFFER)
 }
 
 const onAcceptOffer = async (caller: string) => {
-  await submit(caller, ShoppingActions.ACCEPT_OFFER, refetch)
+  await submit(caller, ShoppingActions.ACCEPT_OFFER)
 }
```

Comment:
Why remove?

Reply:
> Why remove?

Just to fix the warning from deep scan. The `submit` function does not actually consume the `refetch` function.

Reply:
Yup thanks, I spotted bug #5880